tests/durability/test_disk_failure_recovery.py

```diff
@@ -520,3 +521,460 @@ def test_sns_repair_fail_disk_diff_cvg_equal_to_k(self):
 
         LOGGER.info("COMPLETED: Test SNS repair works fine with failed disks "
                     "are equal to K(parity units) and from different cvg")
+
+    # pylint: disable=too-many-statements
+    @pytest.mark.data_durability
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-36396")
+    def test_near_full_sns_repair_fail_disk_diff_cvg_less_than_k(self):
```

Comment:
can we shorten the title? Maybe we can remove sns_repair as this test suite does the same?

Reply:
Done